tests/themispp/.clang-format

```diff
@@ -0,0 +1,19 @@
+﻿---
+BasedOnStyle: LLVM
+AccessModifierOffset: -4
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: false
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: false
+AllowShortIfStatementsOnASingleLine: false
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakTemplateDeclarations: true
+BreakBeforeBraces: Linux
+BreakConstructorInitializersBeforeComma: true
+ColumnLimit: 120
```

Comment:
other files have 100?

Reply:
Yes. That's the only difference between this style and the main one.

Maybe we should just go for 120 for all of them as it looks a bit better with the current code base. I feel bad for making a wrong guess with that 100.